Doc/library/stdtypes.rst

```diff
@@ -1911,7 +1911,14 @@ expression support in the :mod:`re` module).
    Return the lowest index in the string where substring *sub* is found within
    the slice ``s[start:end]``.  Optional arguments *start* and *end* are
    interpreted as in slice notation.  Return ``-1`` if *sub* is not found.
+   For example::
+
+      >>> 'spam, spam, spam'.find('sp')
+      0
+      >>> 'spam, spam, spam'.find('sp', 5)
+      6
 
+   See also :meth:`rfind` and :meth:`index`.
```

Comment:
A newline:
```suggestion
   See also :meth:`rfind` and :meth:`index`.

```
Should [fix](https://github.com/python/cpython/actions/runs/17259042588/job/48976392572?pr=134529):

> /home/runner/work/cpython/cpython/Doc/library/stdtypes.rst:1924: WARNING: Lexing literal_block "The :meth:`~str.find` method should be used only if you need to know the\nposition of *sub*.  To check if *sub* is a substring or not, use the\n:keyword:`in` operator::\n\n   >>> 'Py' in 'Python'\n   True" as "python3" resulted in an error at token: '`'. Retrying in relaxed mode. [misc.highlighting_failure]